### PR TITLE
tektoncd-pipeline: disable CI on master and add release-next

### PR DESCRIPTION
**Pull request by vdemeester**
- Disable `master` (as for `openshift/knative-*`)
- Add a `release-next` branch that track upstream master

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

**Comment by vdemeester** [comment]
/hold
cc @stevekuznetsov something is weird with the generated config (see https://github.com/openshift/release/pull/3442/commits/fece1c896a7494ae3a629b74e654bd4b6755e8eb which is a commit from the `docker run …` command)

**Comment by vdemeester** [comment]
/hold cancel

nevermind, my image wasn't up-to-date :sweat_smile:

File: ci-operator/jobs/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-next-presubmits.yaml [hunk 1]
```diff
@@ -81,8 +81,8 @@ presubmits:
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
-              key: openshift-tektoncd-pipeline-master.yaml
-              name: ci-operator-master-configs
+              key: openshift-tektoncd-pipeline-release-next.yaml
+              name: ci-operator-misc-configs
```
Comment (matzew) [inline ci-operator/jobs/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-next-presubmits.yaml]: :+1:

**Pull request merged**